pyEpiabm/pyEpiabm/sweep/initial_place_sweep.py

```diff
@@ -67,10 +75,14 @@ def __call__(self, *args):
                                               power_law_params=power_list)
 
                 elif place.place_type.name == "CareHome":
+                    if person_list is not None:
+                        person_list_use = person_list.copy()
+                    else:
+                        person_list_use = None
                     helper.update_place_group(
                         place,
                         group_size=ave_group_size,
-                        person_list=person_list.copy(),
+                        person_list=person_list_use,
                         person_weights=weights,
```

Comment:
I'm slightly what is going on here with all the default commands. What happens if these arguments aren't specified (it looks like no one is added?) When does the `person_list` default to None?

Reply:
If no care home parameter values are specified then care homes were being created with the person list from the previous place (because of person_list.copy() and the fact that in the absence of parameters create_age_weights() never gets called). So values will now default when no care home parameters are provided such that no people are added to care homes.

Reply:
When person_list=None in update_place_group, it looks like it is defaulted to all people in the cell. It looks like it will still work as intended because you also have mean_capacity=0 in that case, but would it be more clear to have person_list_use = [] instead of None?